### PR TITLE
Remove EA banner from step up authn and oidc pages

### PR DESCRIPTION
**Pull request by nakuljoshi-okta**
## Description:
- **What's changed?** 
- Remove EA banner from step up authentication and oidc pages
- **Is this PR related to a Monolith release?**  `2022.11.0`

### Resolves:

* [OKTA-544556](https://oktainc.atlassian.net/browse/OKTA-544556)

### CODE REVIEW

**Review by mymoonasaifunnissameerasahib-okta — APPROVED** [review]

**Review by IanHakes-okta — APPROVED** [review]
LGTM

**Comment by larsjohansen-okta** [comment]
I wonder if we can make this section a bit more clear:

```
In Okta Classic Engine when a user doesn't have a session, the more restrictive policy is evaluated first. If the Okta sign-on policy is more restrictive, then that is evaluated first. Otherwise, the application sign-on policy is evaluated. When an ACR value is passed in the authorize request and the application sign-on policy is more restrictive, the application sign-on policy is evaluated. If it isn’t more restrictive, then the acr_values parameter in the request is evaluated. When a user already has a session, the application sign-on policy is always applied first. Then, if the application sign-on policy requirements are satisfied, the acr_values parameter in the request is evaluated.
```

Specifically, this is not clear whether you talk about the acr_values or Okta SoP: 
```When an ACR value is passed in the authorize request and **the application sign-on policy is more restrictive**, the application sign-on policy is evaluated. If it isn’t more restrictive, then the acr_values parameter in the request is evaluated.
```

Can we make it clear?

**Comment by larsjohansen-okta** [comment]
Sorry for last minute requests, but should we not group the 1fa values together in the table? Seems weird to do this order 1fa, 2fa, 1fa:pwd then 2fa:if:possible.

image

**Comment by larsjohansen-okta** [comment]
image
Typo, should be `{yourOktaDomain}` and not `{yourOktadomain}`.

**Comment by larsjohansen-okta** [comment]
image
Can we remove the `<` and `>` before and after the reference to jwt.io?

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Explore the OpenID Connect & OAuth 2.0 API: [![Run in Postman](https://run.pstmn
 | --------------------------------------------------------------------------------  | -------------------------------------------------------------------------               |
 | [/authorize](#authorize)                                                          | Interact with the resource owner and obtain an authorization grant.                     |
 | [/par](#par)                                                                      | Push an authorization request payload directly to the authorization server that responds with a request URI value for use in subsequent authorization requests to the `/authorize` endpoint. <ApiLifecycle access="ie" />                     |
-| [/device/authorize](#device-authorize)                                            | Obtain an activation code for the resource owner.<ApiLifecycle access="ea" />           |
+| [/device/authorize](#device-authorize)                                            | Obtain an activation code for the resource owner.                                       |
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:28]: 👍🏻 We must've missed it

**Comment by nakuljoshi-okta** [comment]
> I wonder if we can make this section a bit more clear:
> 
> `In Okta Classic Engine when a user doesn't have a session, the more restrictive policy is evaluated first. If the Okta sign-on policy is more restrictive, then that is evaluated first. Otherwise, the application sign-on policy is evaluated. When an ACR value is passed in the authorize request and the application sign-on policy is more restrictive, the application sign-on policy is evaluated. If it isn’t more restrictive, then the acr_values parameter in the request is evaluated. When a user already has a session, the application sign-on policy is always applied first. Then, if the application sign-on policy requirements are satisfied, the acr_values parameter in the request is evaluated.`
> 
> Specifically, this is not clear whether you talk about the acr_values or Okta SoP in the section about no session: `When an ACR value is passed in the authorize request and **the application sign-on policy is more restrictive**, the application sign-on policy is evaluated. If it isn’t more restrictive, then the acr_values parameter in the request is evaluated.`
> 
> Can we make it clear?
> 
> Is my understanding right like this for when you do not have a session:
> 
> * If OSoP is more restrictive than ASoP, then evaluate OSoP first, then if acr_values is more restrictive than ASoP evaluate acr_values after OSoP and then finally evaluate ASoP.
> * If ASoP is more restrictive than OSoP, check if acr_values is more restrictive than ASoP and, if so, evaluate acr_values first, then ASoP and then finally OSoP. If acr_values is not more restrictive than ASoP, evaluate ASoP, then acr_values and then finally OSoP.
> 
> It might even be better with a table/diagram to understand this well; it's somewhat complex.

Yes your understanding is correct. I think we first compare OSoP and ASoP in this sentence
`If the Okta sign-on policy is more restrictive, then that is evaluated first.`
and then the next sentence is in continuation and we compare acr and ASoP (it is assumed that OSoP is evaluated before)
`When an ACR value is passed in the authorize request and the application sign-on policy is more restrictive, the application sign-on policy is evaluated. If it isn’t more restrictive, then the acr_values parameter in the request is evaluated.`.

Basically it's in order of restrictiveness of policy. But following logic applies multiple times till you satisfy all the factors.
```
if (OSoP > ASoP)
 OSoP
else
  if (ACR > ASoP)
    ACR
  else 
    ASoP
```

Any suggestion on wording?

**Review by susanharper-okta — APPROVED** [review]
LGTM
